ajax/taskjob_logs.php

```diff
@@ -46,7 +46,6 @@
 }
 
 //unlock session since access checks have been done
-session_write_close();
```

Comment:
Ah ok, and you haven't problems on monitoring logs (with many task logs) ?
Because session block page loading when other page not finished (except if session write lock released)